ThunderCats/UITextField+TCAdditions.m

```diff
@@ -40,4 +40,27 @@ - (void)tc_setLeftAndRightPadding:(CGFloat)paddingWidth
     self.rightViewMode = UITextFieldViewModeUnlessEditing;
 }
 
+- (void)tc_addToolbarWithNextButton:(UITextField * __tc_nonnull)nextTextField
```

Comment:
Yea, I wrote the original version of this in Swift and the next text field had a named parameter so it was clear:
```swift
func addToolbarWithNextButton(nextTextField nextTextField: UITextField)
```

I can't think of a concise and clear why to write it for Objective-C. I mean, `addToolbarWithNextButtonForNextTextField:` is a little verbose, but it's fairly clear.